.config/CredScanSuppressions.json

```diff
@@ -32,6 +32,10 @@
         {
             "file": "\\toolkit\\tools\\pkg\\imagecustomizerlib\\testdata\\addusers-config.yaml",
             "_justification": "Dummy secrets used to unit test configuration code."
+        },
+        {
+            "file": "\\toolkit\\tools\\pkg\\osmodifierlib\\testdata\\addusers-config.yaml",
+            "_justification": "Dummy secrets used to unit test configuration code."
```

Comment:
Is there a reason we cannot remove this code?

Reply:
there is a fake password hash inside the test file, and Pawel pointed out it triggered the [credentials scan to fail](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=485517&view=logs&j=096205eb-4599-56da-affa-13246cc49eac&t=7e5c3140-9b2b-5395-0e58-9e02fe0de382&l=204439) in some of builds. So this is added to tell the scanner that this is not an actual password.

Reply:
as you mentioned this testdata is also in  `3.0-dev`, should the change also be checked into that branch as well?

Reply:
Yes, that would be great. Please also make the same change against `fasttrack/2.0`. This will unblock our PR checks there.